google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -267,31 +289,47 @@ public HttpHeaders getHeaders() {
     /**
      * Sets the HTTP response headers.
      *
-     * <p>
-     * Overriding is only supported for the purpose of calling the super implementation and changing
-     * the return type, but nothing else.
-     * </p>
+     * <p>Overriding is only supported for the purpose of calling the super implementation and
+     * changing the return type, but nothing else.
      */
     public Builder setHeaders(HttpHeaders headers) {
       this.headers = Preconditions.checkNotNull(headers);
       return this;
     }
 
     /** Returns the HTTP response content or {@code null} for none. */
-    public final String getContent() {
-      return content;
+    public final String getContent() throws IOException {
+      if (content == null || charset == null) {
```

Comment:
why charset == null? if charset is not available it should use default charset.

Reply:
Removed this check